src/openlifu/db/session.py

```diff
@@ -152,5 +153,6 @@ def to_file(self, filename):
 
         :param filename: Name of the file
         """
+        Path(filename).parent.mkdir(exist_ok=True)
```

Comment:
I think that this should be in a separate commit (or maybe modifying commit message to include fix to_file)

Reply:
Good point, thanks